autoflip_calculators/text_detection_calculator.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+package mediapipe.autoflip;
+
+import "mediapipe/examples/desktop/autoflip/quality/visual_scorer.proto";
+import "mediapipe/framework/calculator.proto";
+
+message TextDetectionCalculatorOptions {
+  extend mediapipe.CalculatorOptions {
+    optional TextDetectionCalculatorOptions ext = 284226722;
+  }
+
+  // Options for generating a score for the text from its visual
+  // appearance. The generated score is used to modulate the detection 
+  // scores for text bbox region types.
+  optional VisualScorerOptions scorer_options = 1;
+
+  // If true, generate a score from the appearance of the text and use it to
+  // modulate the detection scores for text bboxes.
+  optional bool use_visual_scorer = 2 [default = true];
```

Comment:
Add comment to specify what happens if use_visual_scorer is set false (the score will be taken from the detector's value).